### PR TITLE
Add Oracle scenario guide

### PR DESCRIPTION
**Pull request by nalsaber**
##### SUMMARY
Add Oracle scenario guide for Oracle Cloud Infrastructure Modules

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
scenario guide

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+************
+Introduction
+************
+
+Oracle provides a number of Ansible modules to interact with Oracle Cloud Infrastructure (OCI). In this guide, we will explain how you can use these modules to orchestrate, provision and configure your infrastructure on OCI. 
+
+************
+Requirements
+************
+In order to use the OCI Ansible modules, you must have the following prerequisites on your controller computer, which is the computer from which Ansible playbooks are executed.
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
To use the OCI Ansible modules, you must have the following prerequisites on your control node, the computer from which Ansible playbooks are executed.
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+1. `An Oracle Cloud Infrastructure account. <https://cloud.oracle.com/en_US/tryit>`_
+
+2. A user created in that account, in a security group with a policy that grants the necessary permissions for working with resources in those compartments. For guidance, see `How Policies Work <https://docs.cloud.oracle.com/iaas/Content/Identity/Concepts/policies.htm>`_.
+3. The necessary credentials and OCID information.
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst:20]:
```suggestion

3. The necessary credentials and OCID information.
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+************
+Installation
+************ 
+1. Install the Oracle Cloud Infrastructure Python SDK (more details `here <https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/installation.html>`_):
+
+.. code-block:: bash
+
+        pip install oci
+
+2.  You can use either of the following two methods to install OCI Ansible Modules:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
2.  Install the Ansible OCI Modules in one of two ways:
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+a.	From Galaxy: 
+
+.. code-block:: bash
+
+ ansible-galaxy install oracle.oci_ansible_modules
+
+b.	From github:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
b.	From GitHub:
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+     $ git clone https://github.com/oracle/oci-ansible-modules.git
+
+.. code-block:: bash
+
+    $ cd oci-ansible-modules
+
+
+Run one of the following commands:
+
+- If Ansible is installed as a user:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
- If Ansible is installed only for your user: 
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+.. code-block:: bash
+
+    $ sudo ./install.py
+
+*************
+Configuration
+*************
+
+When creating and configuring Oracle Cloud Infrastructure resources, Ansible modules use authentication information that is outlined on `this page <https://docs.cloud.oracle.com/iaas/Content/API/Concepts/sdkconfig.htm>`_.
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
When creating and configuring Oracle Cloud Infrastructure resources, Ansible modules use the authentication information outlined `here <https://docs.cloud.oracle.com/iaas/Content/API/Concepts/sdkconfig.htm>`_.
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+.
+ 
+*******
+Samples
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
Examples
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+*******
+Launch a Compute Instance
+=========================
+This `sample playbook <https://github.com/oracle/oci-ansible-modules/tree/master/samples/compute/launch_compute_instance>`_
+shows how to launch a public Compute instance and then access the instance from an Ansible module over an SSH connection. The sample illustrates how to do the following:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
launches a public Compute instance and then accesses the instance from an Ansible module over an SSH connection. The sample illustrates how to:
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+- Generate a temporary, host-specific SSH key pair.
+- Specify the public key from the key pair for connecting to the instance, and then launch the instance.
+- Connect to the newly launched instance using SSH.
+
+Create and Manage Autonomous Data Warehouse
+===========================================
+This `playbook <https://github.com/oracle/oci-ansible-modules/tree/master/samples/database/autonomous_data_warehouse>`_
+shows how to create an Autonomous Data Warehouse and manage its lifecycle. The sample shows how to do the following:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
creates an Autonomous Data Warehouse and manage its lifecycle. The sample shows how to:
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+- Set up an Autonomous Data Warehouse.
+- List all of the Autonomous Data Warehouse instances available in a compartment, filtered by the display name.
+- Get the "facts" for a specified Autonomous Data Warehouse.
+- Stop and start an Autonomous Data Warehouse instance.
+- Delete an Autonomous Data Warehouse instance.
+
+Create and Manage Autonomous Transaction Processing
+===================================================
+This `sample playbook shows <https://github.com/oracle/oci-ansible-modules/tree/master/samples/database/autonomous_database>`_
+  how to create an Autonomous Transaction Processing database and manage its lifecycle. The sample shows how to do the following:
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
  creates an Autonomous Transaction Processing database and manage its lifecycle. The sample shows how to:
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+shows how to create an Autonomous Data Warehouse and manage its lifecycle. The sample shows how to do the following:
+- Set up an Autonomous Data Warehouse.
+- List all of the Autonomous Data Warehouse instances available in a compartment, filtered by the display name.
+- Get the "facts" for a specified Autonomous Data Warehouse.
+- Stop and start an Autonomous Data Warehouse instance.
+- Delete an Autonomous Data Warehouse instance.
+
+Create and Manage Autonomous Transaction Processing
+===================================================
+This `sample playbook shows <https://github.com/oracle/oci-ansible-modules/tree/master/samples/database/autonomous_database>`_
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
This `sample playbook <https://github.com/oracle/oci-ansible-modules/tree/master/samples/database/autonomous_database>`_
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+Configuration
+*************
+
+When creating and configuring Oracle Cloud Infrastructure resources, Ansible modules use authentication information that is outlined on `this page <https://docs.cloud.oracle.com/iaas/Content/API/Concepts/sdkconfig.htm>`_.
+.
+ 
+*******
+Samples
+*******
+Launch a Compute Instance
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]: titles should be Sentence case:
```suggestion
Launch a compute instance
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+=========================
+This `sample playbook <https://github.com/oracle/oci-ansible-modules/tree/master/samples/compute/launch_compute_instance>`_
+shows how to launch a public Compute instance and then access the instance from an Ansible module over an SSH connection. The sample illustrates how to do the following:
+
+- Generate a temporary, host-specific SSH key pair.
+- Specify the public key from the key pair for connecting to the instance, and then launch the instance.
+- Connect to the newly launched instance using SSH.
+
+Create and Manage Autonomous Data Warehouse
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
Create and manage Autonomous Data Warehouses
```

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+===========================================
+This `playbook <https://github.com/oracle/oci-ansible-modules/tree/master/samples/database/autonomous_data_warehouse>`_
+shows how to create an Autonomous Data Warehouse and manage its lifecycle. The sample shows how to do the following:
+- Set up an Autonomous Data Warehouse.
+- List all of the Autonomous Data Warehouse instances available in a compartment, filtered by the display name.
+- Get the "facts" for a specified Autonomous Data Warehouse.
+- Stop and start an Autonomous Data Warehouse instance.
+- Delete an Autonomous Data Warehouse instance.
+
+Create and Manage Autonomous Transaction Processing
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
Create and manage Autonomous Transaction Processing
```

**Review by acozine — COMMENTED** [review]
Thanks for the content @nalsaber - I made some suggestions. Our aim is to make the documentation concise, direct, and simple. See the Ansible [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html) for more details.

File: docs/docsite/rst/scenario_guides/guide_oracle.rst [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+1. `An Oracle Cloud Infrastructure account. <https://cloud.oracle.com/en_US/tryit>`_
+
+2. A user created in that account, in a security group with a policy that grants the necessary permissions for working with resources in those compartments. For guidance, see `How Policies Work <https://docs.cloud.oracle.com/iaas/Content/Identity/Concepts/policies.htm>`_.
+
+3. The necessary credentials and OCID information.
+
+************
+Installation
+************ 
+1. Install the Oracle Cloud Infrastructure Python SDK (more details `here <https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/installation.html>`_):
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_oracle.rst]:
```suggestion
1. Install the Oracle Cloud Infrastructure Python SDK (`detailed installation instructions <https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/installation.html>`_):
```

**Comment by nalsaber** [comment]
@acozine Thanks for  your help!

**Pull request merged**